src/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url("./images/star.svg") no-repeat center;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
```

Comment:
group this selectors like that 
 ` 
 .block1, .block2, .block3 {
   color: white
 }
`

Reply:
I know that it's correct to group like that, but I receive errors when I run lint. 
src/style.css
 16:35  ✖  Expected newline after ","   selector-list-comma-newline-after
 18:39  ✖  Expected newline after ","   selector-list-comma-newline-after
 20:39  ✖  Expected newline after ","   selector-list-comma-newline-after
So, I left it as it is.

Reply:
linter wants you to put each new selector on a new line
.block1,
.block2,
.block3 {
  ....
}